File: Data Structures & Algorithms/find-duplicate-integer/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    findDuplicate(nums) {
        let map = new Map()
        for (let i of nums)
        {
            if(map.has(i))
            {
                return i
            }
            else {
                map.set(i, 0)
            }
        }
        
    }
}

File: Data Structures & Algorithms/duplicate-integer/submission-0.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        hash_map = dict()
        for i in nums:
            if(hash_map.get(i) == None):
                hash_map[i] = 1
            else:
                return True
        
        return False

        